### DISCUSSION TITLE
Elementor pro widgets activity selector

### DISCUSSION THREAD

**Discussion by NAROLLES** [elementor/elementor#15287] [Feature Request]
Hello
To optimize the plugin, some widgets may never be used depending on the project.
Would it be possible to provide a selector that would make a widget inactive and therefore lighten the plugin on loading

**Comment by aaliyahfraser** [comment]
Thanks for this article. You have shown some excellent plugins in this article. All the plugins are perfect. I have also posted a blog on this topic. This is because Elementor has so many free amazing add-ons. Easy-to-use features made creating websites a breeze. Here we talk about “Best Elementor Addons and Plugins for WordPress 2023 – Compared” You can check out it at – https://quomodosoft.com/best-elementor-addons-and-plugins/
